helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -26,6 +26,7 @@ import org.ossreviewtoolkit.model.config.PathExcludeReason
 import org.ossreviewtoolkit.model.config.PathExcludeReason.BUILD_TOOL_OF
 import org.ossreviewtoolkit.model.config.PathExcludeReason.DOCUMENTATION_OF
 import org.ossreviewtoolkit.model.config.PathExcludeReason.TEST_OF
+import org.ossreviewtoolkit.utils.common.FileMatcher
```

Comment:
Commit message:

> Compared to regular expression

It still confuses me to refer to regular expressions here, because to the reader there never was the alternative to use regular expressions instead of globs; you straight introduce globs without having had regular expressions before.

How about something like this:

> Make the search in the project's source tree for paths to be excluded more flexible by using globs. Any matching directory paths are then added to the path excludes with a `/**` suffix.

Additionally, it would be nice if you could explain why you're not simply writing out these globs as-is to the path excludes. My *guess* is that you want to keep path excludes to a minimum and only list patterns that actually match something in the project's source tree instead of always listing a (big) list of default excludes. So maybe also say:

> Note that these globs are expanded at the runtime of the helper command in order to create a minimal set of path excludes that actually match paths in the project's source tree. So, while these globs could also haven been used as path exclude patterns as-is, this is not done to avoid having patterns that do not match anything.